common/contract.ts

```diff
@@ -107,6 +111,21 @@ export type CPMM = {
   }
 }
 
+export type Uniswap2 = {
+  mechanism: 'uniswap-2'
+  // outcome can be e.g. 'M$' or a 'SHARE'
+  pool: { [outcome: string]: number }
+  // Unused; just for making Typescript happy, because code paths
+  // assume this thing falls under DPM-2
```

Comment:
There must be a way to get typescript to not complain. Adding an extra field is bad.

Reply:
Yeah, will look into getting rid of the field!